microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md

```diff
@@ -92,12 +92,11 @@ If you have previously turned on per person MFA, you must turn it off before ena
 1. On the multi-factor authentication page, select each user and set their Multi-Factor auth status to **Disabled**.
 
 ## Next steps
-
 - [How to register for their additional verification method](https://support.microsoft.com/office/ace1d096-61e5-449b-a875-58eb3d74de14)
+- [What is: Multifactor Authentication](https://support.microsoft.com/en-us/help/4577374/what-is-multifactor-authentication)
```

Comment:
```suggestion
- [What is: Multifactor Authentication](https://support.microsoft.com/en-us/help/4577374/what-is-multifactor-authentication)
```

Reply:
Thank you.I have removed the locale from the URL